src/client/components/input/drag-and-drop-image.js

```diff
@@ -1,11 +1,25 @@
 const React = require('react');
 const Input = require('react-bootstrap').Input;
 
+/**
+* This class is derived from the React Component base class to render the
+* drag and drop image functionality.
```

Comment:
"to render an image which supports drag and drop functionality" sounds better here